requirements.txt

```diff
@@ -8,3 +8,8 @@ SPARQLWrapper>=1.8.5
 websockets>=8.1
 python-dotenv>=0.13.0
 trompace-client
+Wikidata>=0.7.0
+redis>=3.5.3
+redis-decorato>=0.4
```

Comment:
check this - is it supposed to be `redis-decorator`?

Reply:
thanks!